src/main/java/Todo.java

```diff
@@ -0,0 +1,25 @@
+public class Todo extends Task {
+    public Todo (String description) throws EmptyDescriptionException {
```

Comment:
There's an extra whitespace after `Todo` here, should be `public Todo(String description)`. Similar issue in the other overloaded constructor.